concrete/src/Error/ErrorList/Error/AbstractError.php

```diff
@@ -1,45 +1,100 @@
 <?php
+
 namespace Concrete\Core\Error\ErrorList\Error;
 
 use Concrete\Core\Error\ErrorList\Field\FieldInterface;
 
-abstract class AbstractError implements ErrorInterface
+abstract class AbstractError implements HtmlAwareErrorInterface
 {
-
     /**
-     * @var FieldInterface
+     * The field associated to the error.
+     *
+     * @var \Concrete\Core\Error\ErrorList\Field\FieldInterface
```

Comment:
Does it make sense that there is a distinction between this phpdoc and this https://github.com/concrete5/concrete5/pull/7375/files#diff-e3cceba30cf0f334a49ec57aa954d591R38?

Reply:
Well, not at all :+1: